db/seeds.rb

```diff
@@ -1,199 +1,72 @@
-                                      While this snowboard is gripped and ready to rip, it is not recommended for riding at resorts or on hardpack or icy conditions.
-                                      Please consult your resort before riding The Throwback on their trails.',
-                        price: 325,
-                        brand: 'Burton',
-                        image: File.open('app/assets/images/neighborhood_throwback.jpeg')
-                      }])
+  def generate_users
+    50.times do |i|
+      user = User.create!(username: Faker::Internet.user_name,
+                          password: Faker::Internet.password)
 
-boots = Category.find_by(title: 'Boots')
-        .items
-        .create([{ name: 'cool boots',
-                   description: 'These boots were made for shreddin',
-                   price: 200,
-                   brand: 'Ride',
-                   image: File.open('app/assets/images/boots1.jpg')
-               },
+      puts "User #{i}: #{user.username} created!"
+    end
+  end
 
-                 { name: 'gnar kicks',
-                   description: 'Foot-tappin-good boots. These puppies will keep you warm.',
-                   price: 150,
-                   brand: 'Thirty-Two',
-                   image: File.open('app/assets/images/boot2.jpg')
-                 },
-                 { name: 'TRIAD BOOTS',
-                   description: 'The Triad strikes a chord with freestyle riders as the go to performance snowboard boot built to last. Featuring the Tongue Tied Boa
-                                 closure system for next level heel hold and maximum control on a traditional lace snowboard boot. Blending top technologies such as
-                                 the all new FlexLight outsole for better cushioning and custom fitting Intuition Mobile Foam Liner, these lightweight kicks prove
-                                 durable and supportive, park lap after park lap.',
-                   price: 230,
-                   brand: 'Ride',
-                   image: File.open('app/assets/images/triad_boots.png')
-                 },
-                 { name: 'HASHTAG BOOT',
-                   description: 'K2’s Fast-In technology keeps things simple and quick so more time is spent actually snowboarding rather than fighting with a boot
-                                 that would rather stay home.',
-                   price: 170,
-                   brand: 'K2',
-                   image: File.open('app/assets/images/hashtag.png')
-                 }])
+  def generate_items
+    store = Store.create!(name: "Cars-o-rama")
+    500.times do |i|
+      car = store.cars.create!(make: Faker::Commerce.product_name,
+                               model: Faker::Name.first_name,
+                               year: Faker::Number.between(from = 1900, to = 2015),
```

Comment:
Useless assignment to variable - `from`.<br>Useless assignment to variable - `to`.<br>Line is too long. [83/80]